pkg/controller/controller_utils.go

```diff
@@ -905,6 +905,15 @@ func PatchNodeTaints(c clientset.Interface, nodeName string, oldNode *v1.Node, n
 	return err
 }
 
+// UpdateNodeTaints updates node's taints.
+func UpdateNodeTaints(c clientset.Interface, nodeName string, oldNode *v1.Node, newNode *v1.Node) error {
```

Comment:
Can we please add more comments around what exactly are these [set of] methods for?
- nodeName in signature but not used inside- may be for uniformity?